requirements.txt

```diff
@@ -35,8 +44,11 @@ redis==2.10.3
 requests==2.13.0
 requests-oauthlib==0.5.0
 restable>=0.1.3
+singledispatch==3.4.0.3
+six==1.10.0
+static3==0.7.0
+tornado==4.5.3
```

Comment:
do you need all these new libraries?

Reply:
Nope I do not believe Jean Bacon required any extra libraries.